test/common/config/subscription_factory_impl_test.cc

```diff
@@ -314,6 +314,34 @@ TEST_F(SubscriptionFactoryTest, HttpSubscriptionNoRefreshDelay) {
                             "refresh_delay is required for REST API configuration sources");
 }
 
+TEST_F(SubscriptionFactoryTest, GrpcSubscriptionInvalidRetryConfig) {
```

Comment:
nit: please add a short comment before each test (here and in other tests) that explains what the test validates.

Reply:
done